index.js

```diff
@@ -0,0 +1,40 @@
+import * as FN from './modules/functions.js';
+import { Book, bookList, bookListSection } from './modules/classes.js';
+import { DateTime } from './node_modules/luxon/build/es6/luxon.js';
```

Comment:
- [ ] Good job so far :clap:  But From the project requirements you should display the date and time below the navigation bar and also use luxon.js by downloading the package and adding it to your project as a JavaScript file. Please make re to follow the instructions below :arrow_down: in order to make your project as expected :+1: 
![luxoooooooooooooooon111ÉÉÉ](https://user-images.githubusercontent.com/6615394/209450579-5dccdc86-9bc5-40dc-88bb-2870575159c0.jpg)